src/app/components/ImageWithCaption/index.tsx

```diff
@@ -33,9 +31,9 @@ const shouldLazyLoad = (position: number[]) =>
 const renderCopyright = (copyright: string) =>
   copyright && <Copyright>{copyright}</Copyright>;
 
-const renderCaption = (block: object, type: string, service: Services) =>
+const renderCaption = (block: object, type: string) =>
   // @ts-expect-error - TODO: fix types for blocks
-  block && <Caption block={block} type={type} service={service} />;
+  block && <Caption block={block} type={type} />;
```

Comment:
Caption looks like it never needed service in the first place, right?

Reply:
Yea its using the new Caption component with the new component standards, so will get service font sizing etc from the ThemeProvider directly